x-pack/plugins/cases/public/components/actions/copy_id/translations.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const COPY_ID_ACTION_LABEL = i18n.translate('xpack.cases.actions.copyIDSingleCase', {
```

Comment:
The two translation files seem identical. We could import the translations from `x-pack/plugins/cases/public/common/translations.ts` in the actions.